Source/WebCore/rendering/svg/RenderSVGContainer.cpp

```diff
@@ -91,7 +91,7 @@ void RenderSVGContainer::layout()
 void RenderSVGContainer::layoutChildren()
 {
     SVGContainerLayout containerLayout(*this);
-    containerLayout.layoutChildren(selfNeedsLayout() || SVGRenderSupport::filtersForceContainerLayout(*this));
```

Comment:
Why can filtersForceContainerLayout be removed?

Reply:
In LBSE, no resource should be responsible for re-layouts, filters need another less-intrusive mechanism, and I propose to take care about that when we implement filters.

Reply:
That makes sense, I missed the fact it is removed from LBSE classes but not from legacy.